main.c

```diff
@@ -137,9 +137,31 @@ void hexdump(const void *memory, size_t length)
 }
 #else
 static void print(const char * unused) { }
+static void print_p(const void * unused) { }
 static void hexdump(const void *unused, size_t unused2) { }
 #endif
 
+static void extend_pcr(struct tpm *tpm, void *data, u32 size, u32 pcr)
```

Comment:
Would it be possible to move this change to its own commit since this part of the change is not related to using the new kernel info structure?

Reply:
Will a separate commit be enough or should I make another PR?

Reply:
Earlier I would have said no, but you actually have this change in your MB2 PR as well. Since no one has made any comments in either PR, maybe a quick PR with just this change and we get it merged quickly. Then you can rebase both PRs afterwards and drop the change from both. That sound acceptable?

Reply:
Done